.github/workflows/docker-build.yaml

```diff
@@ -304,6 +304,33 @@ jobs:
             -   name: Nginx container logs
                 if: ${{ failure() }}
                 run: docker compose logs webserver
+    unit-test-storefront:
```

Comment:
The same test should also be added to `build-fork-docker-images` job in the same file. 
After `Check storefront standards` please add 
```
-   name: Run storefront unit tests
     run: docker compose exec storefront pnpm test--no-watch
```